### ISSUE TITLE
Fiche produit : produit favoris

### ISSUE THREAD

**Issue by Karramiel** [Karramiel/LPC#31]
Possibilité de mettre des produits en favoris (champs perso)

**Comment by Karramiel** [comment]
La mise en favoris (Coeur) dans les fiches produit ne fonctionne pas.

**Comment by ghost** [comment]
ça été déjà corrigé non?

**Comment by Karramiel** [comment]
non, c'est les favoris dans les FICHES produite mais c'est pour moi je pense ce ticket (php un peu complexe)
